src/TLuaInterpreter.cpp

```diff
@@ -6303,10 +6303,6 @@ int TLuaInterpreter::setLink(lua_State* L)
 
     auto console = CONSOLE(L, windowName);
     console->setLink(_linkFunction, _linkHint);
-    if (console != host.mpConsole) {
```

Comment:
What has this to do with the subject of this PR? 😕

Reply:
Nothing, I forgot to switch workspaces.

Reply:
Reverted for now.